content/03.methods.md

```diff
@@ -83,19 +83,19 @@ We used GATK Haplotype caller.
 #### SNV and INDEL calling
 
 We used four variant callers to call SNVs and INDELS from targeted DNA panel, WXS, and WGS data: Strelka2, Mutect2, Lancet, and VarDict.
-The input interval BED files for both panel and WXS data were padded by 100 bp on each side for all variant calling algorithm runs.
-The BED files for WGS were not padded for Mutect2 and Strelka2 runs, were padded by 300 bp on each side for Lancet, and by 100 bp on each side for VarDict.
+The input interval BED files for both panel and WXS data, derived from probes used to create the assays, were padded by 100 bp on each side for all variant calling algorithm runs.
+We utilized the non-padded BROAD Institute interval calling list `wgs_calling_regions.hg38.interval_list`, comprised of the full genome minus N bases, as the WGS input BED, unless otherwise noted below. 
 Strelka2 [@doi:10/gdwrp4] v2.9.3 was run using default parameters on human genome reference hg38, canonical chromosomes only (chr1-22, X,Y,M), as recommended by the authors.
```

Comment:
Just to clarify, this is also using the BROAD intervals? Having this sentence after the discussion of the intervals makes it seem like it might have been run on the whole genome, without that filtering.

Reply:
@jashapiro are you asking about the panel and WXS intervals or WGS? The first sentence is now about panel/WXS, which were derived from probes used to create the assays (these are always from a manufacturer of the kit). We also provided these in the data release. We can't link to them unless we link to the S3 data (the TGEN panel BED was from an earlier manufacturer and we had to get permission to use, so it is not public, except for our release). The second one states the BROAD intervals were used for WGS. Let me know what you think.

Reply:
I think I was confused by the repeated reference to hg38 here. Perhaps just: 
```suggestion
Strelka2 [@doi:10/gdwrp4] v2.9.3 was run using default parameters on canonical chromosomes only (chr1-22, X,Y,M), as recommended by the authors.
```

Reply:
how about `Strelka2 [@doi:10/gdwrp4] v2.9.3 was run using default parameters for canonical chromosomes (chr1-22, X,Y,M) of human genome reference hg38, as recommended by the authors.` so that we have hg38 noted?

Reply:
Everything is hg38, right? Its not specific to strelka...

Reply:
yes, correct... looks like we have inconsistencies - listing it many places

Reply:
removed since we have it in the alignment section